library/cartridge_get_package_info.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+
+import os
+import subprocess
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ansible.module_utils.helpers import ModuleRes
+
+
+argument_spec = {
+    'app_name': {'required': False, 'type': 'str'},
+    'package_path': {'required': True, 'type': 'str'},
+}
+
+
+DEB = 'deb'
+RPM = 'rpm'
+
+
+def run_command_and_get_output(cmd):
+    process = subprocess.Popen(
+        cmd,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE
+    )
+
+    stdout, _ = process.communicate()
+    stdout = stdout.decode('utf-8')
+
+    return process.returncode, stdout
+
+
+def get_package_type(package_path):
+    _, ext = os.path.splitext(package_path)
+    if ext == '.rpm':
+        return RPM
+    elif ext == '.deb':
+        return DEB
+    raise Exception("Unknown package extension: %s" % ext)
+
+
+def get_rpm_info(package_path):
+    # package name
+    cmd = ['rpm', '-qip', package_path]
+    rc, output = run_command_and_get_output(cmd)
+    if rc != 0:
+        raise Exception("failed to get RPM package info: %s" % output)
+
+    m = re.search(r'Name\s*:\s*(.*)\s+', output)
```

Comment:
`.*` can catch any spaces, line ends, etc. Are you sure that this regex is correct? It may be better to use `[^\s]+` or `[^\n]+`. A similar question to other places with `.*` regular expression.

Reply:
Replaced via `\S+`.

Reply:
Are you sure the name cannot contain spaces?

Reply:
Are you sure the name cannot contain spaces?
If in some cases spaces can be in the name, then let's use `[^\n]+`.
If, in all cases, there are no spaces in the application name, this is fine.